### ISSUE TITLE
Request: Equivalent of Yaourt's pacdiffviewer (-C)

### ISSUE THREAD

**Issue by AndydeCleyre** [Jguer/yay#4]
`yaourt -C`, or `sudo pacdiffviewer`, checks the system for `.pacorig`, `.pacnew`, `.pacsave`, and orphaned files. If found, it facilitates deleting, merging (with a user-set mergetool), or replacing relevant files.

The script is available [on github](https://github.com/archlinuxfr/yaourt/blob/master/src/pacdiffviewer.sh.in).

The most important type of file from those listed above is the `.pacnew` class.

**Comment by Jguer** [comment]
I've had that in the TODO list for some time now, still haven't decided on how to implement it
https://github.com/Jguer/yay/projects/1

**Comment by Morganamilo** [comment]
The `pacdiff` command does a similar thing to this. It looks to be exactly the same in functionality. Close this in favour of that?

**Comment by Jguer** [comment]
Yes it's close enough, most we can do is provide an alias for it, but meh
